### ISSUE TITLE
Add DeepAlpha - AI crypto trading bot for DeFi

### ISSUE THREAD

**Issue by stefanoviana** [OffcierCia/DeFi-Developer-Road-Map#138]
Suggesting [DeepAlpha](https://github.com/stefanoviana/deepalpha) for **Tools > Trading**.

AI-powered crypto trading bot for Bybit & Binance futures. LightGBM + XGBoost ensemble, 70.9% accuracy, 72 ML features. MIT licensed, open source.
